RNA-seq/00b-additional_approaches.md

```diff
@@ -5,7 +5,17 @@ As a result, we do not cover every tool or method that would complement what is
 For your reference, we've put together what we might teach if we had more time or if they required fewer computational resources.
 This is intended to be an introduction to these approaches and a "jumping off point" for further reading or experimentation.
 We present them in order of increasing difficulty and/or departure from what is presented in training.
- 
+
+### Link to pre-prepared datasets
```

Comment:
I don't think we intend for `link-data.sh` to be publicly facing and we may not provide `scripts` to participants. Looking at [contributing](https://github.com/AlexsLemonade/training-modules/blob/master/CONTRIBUTING.md#linking-shared-files), my interpretation of:

> Because this script is also used to set up directories for training

is that we will use this script to prep folders that go into `/shared/skel` (https://github.com/AlexsLemonade/training-admin/issues/447). I expect that participants will already have access to anything that calling `link-data.sh` would get them.

The comment on #358

> we will likely want to reference that more files will be available to training participants (look at `scripts/link-data.sh` or rather run it and see what's in `RNA-seq/data`!)

is a comment for us/the person implementing it. Are there QC files that we will not cover during instruction that folks will have access to? We should make sure to mention them here. (My initial recommendation would be in the `Running MultiQC on the gastric cancer samples` section.)

Reply:
> I don't think we intend for `link-data.sh` to be publicly facing and we may not provide `scripts` to participants. Looking at [contributing](https://github.com/AlexsLemonade/training-modules/blob/master/CONTRIBUTING.md#linking-shared-files), my interpretation of:
> 
> > Because this script is also used to set up directories for training
> 
> is that we will use this script to prep folders that go into `/shared/skel` ([AlexsLemonade/training-admin#447](https://github.com/AlexsLemonade/training-admin/issues/447)). I expect that participants will already have access to anything that calling `link-data.sh` would get them.

Yes, this was my intent. I do not think we should provide `scripts` to users (and probably not `setup` folders, though they can look at them on the website)?

Reply:
Gotcha. Yeah I didn't understand that part. Well that makes that part a whole lot shorter! :+1: